c2corg_ui/templates/helpers/view.html

```diff
@@ -497,3 +502,64 @@ <h3 class="heading closed show-phone" ng-click="mainCtrl.animateHeaderIcon($even
       
   % endif
 </%def>
+
+
+
+<%def name="photoswipe_gallery()">\
+  <!-- Root element of PhotoSwipe. Must have class pswp. -->
```

Comment:
Please use mako comment tags ``##`` instead of HTML comments => this way there are not displayed in the end HTML page.